d2bs/kolbot/libs/common/Pickit.js

```diff
@@ -119,45 +119,54 @@ var Pickit = {
 				// Check if the item should be picked
 				status = this.checkItem(pickList[0]);
 
-				if (status.result && this.canPick(pickList[0]) && Item.autoEquipCheck(pickList[0])) {
-					// Override canFit for scrolls, potions and gold
-					canFit = Storage.Inventory.CanFit(pickList[0]) || [4, 22, 76, 77, 78].indexOf(pickList[0].itemType) > -1;
-
-					// Try to make room with FieldID
-					if (!canFit && Config.FieldID && Town.fieldID()) {
-						canFit = Storage.Inventory.CanFit(pickList[0]) || [4, 22, 76, 77, 78].indexOf(pickList[0].itemType) > -1;
-					}
-
-					// Try to make room by selling items in town
-					if (!canFit) {
-						// Check if any of the current inventory items can be stashed or need to be identified and eventually sold to make room
-						if (this.canMakeRoom()) {
-							print("ÿc7Trying to make room for " + this.itemColor(pickList[0]) + pickList[0].name);
-
-							// Go to town and do town chores
-							if (Town.visitTown()) {
-								// Recursive check after going to town. We need to remake item list because gids can change.
-								// Called only if room can be made so it shouldn't error out or block anything.
-
-								return this.pickItems();
-							}
-
-							// Town visit failed - abort
-							print("ÿc7Not enough room for " + this.itemColor(pickList[0]) + pickList[0].name);
-
-							return false;
-						}
-
-						// Can't make room - trigger automule
-						Misc.itemLogger("No room for", pickList[0]);
-						print("ÿc7Not enough room for " + this.itemColor(pickList[0]) + pickList[0].name);
-
-						needMule = true;
-					}
-
-					// Item can fit - pick it up
-					if (canFit) {
-						this.pickItem(pickList[0], status.result, status.line);
+				if (status.result && this.canPick(pickList[0]) /*&& Item.autoEquipCheck(pickList[0])*/) {

```

Comment:
Why is the autoEquipCheck commented out? Is it not compatible with this change?

Reply:
Nice catch. I commented this out because it was redundant with `status.result` as my local copy of Pickit.checkItem() returns a non-zero for items passing Item.autoEquipCheck

I'll uncomment that for this PR. If I get around to pushing my non-sort-related pickit changes, I'll revisit this line there.